src/main/java/com/amazonaws/kinesisvideo/internal/service/AckConsumer.java

```diff
@@ -72,6 +73,11 @@ private void processAckInputStream() {
                     closed = true;
                 } else if (bytesRead != 0) {
                     log.debug("Received ACK bits: {}", bytesString);
+
```

Comment:
Received ACK bits: for streamName=

Reply:
The streamName should  already be in the thread name